pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRBackendHandler.scala

```diff
@@ -1,5 +1,7 @@
 package edu.berkeley.cs.amplab.sparkr
 
+import org.apache.spark.Logging
```

Comment:
The Logging API in Spark is marked as 'do not use outside of Spark` -- so lets stick to `println` for now and we can switch to the logger when we prepare for the merge